app/experimenter/nimbus-ui/src/gql/experiments.ts

```diff
@@ -21,8 +21,8 @@ export const CREATE_EXPERIMENT_MUTATION = gql`
 `;
 
 export const UPDATE_EXPERIMENT_OVERVIEW_MUTATION = gql`
-  mutation updateExperimentOverview($input: UpdateExperimentInput!) {
-    updateExperimentOverview(input: $input) {
+  mutation updateExperimentOverview($input: ExperimentInput!) {
```

Comment:
Might be nice to collapse all these mutations into one on the client-side at some point, but this seems fine enough to minimize impact of the changes for now

Reply:
Yeah I didn't wanna mess with too much structure, but yes that would be interesting.